packages/framework-provider-aws/src/library/read-models-searcher-adapter.ts

```diff
@@ -24,13 +34,28 @@ export async function searchReadModel(
       ExpressionAttributeValues: buildExpressionAttributeValues(filters),
     }
   }
+  if (limit) {
+    params = { ...params, Limit: limit }
+  }
+  if (afterCursor) {
+    params = { ...params, ExclusiveStartKey: afterCursor }
+  }
+
```

Comment:
can't we add this to the initial assignment like this?
```typescript
let params: DocumentClient.ScanInput = {    
    TableName: config.resourceNames.forReadModel(readModelName),    
    ConsistentRead: true,
    Limit: limit,
    ExclusiveStartKey: afterCursor
}
```

If I recall correctly, these params will be `undefined` anyways if they're not specified

Reply:
That's a great suggestion, thanks @juanjoman! I just changed it 🚀